packages/v4/src/content/design-guidelines/demos/card-view/card-view.md

```diff
@@ -0,0 +1,69 @@
+---
+id: Card view
+section: demos
+---
+
+
+A **card view** is a grid of cards in a gallery to facilitate browsing. Card views are typically used to present data set summaries, allowing users to drill down into any card to see more detailed content.
+
+<img src="./img/card-view.png" alt="card view" />
+
+Use a card view to:
+- Display items in a data set that are best identified by a graphic or other visual representation.
+- Make information easier to visualize and compare.
+- Summarize a large number of objects at once within the same page.
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+
+### Background colors for card views
+Card views can be placed on a white or a gray background, depending on your use case.
+
+**Use a white background** if your content view can change, such as when:
+- A card that opens an expandable panel on the right.
+- A card view that toggles into a table or list view.
+
```

Comment:
Adding a comment here to make sure Alana's previous comment/addition doesn't get lost.

She suggested adding "**Note**: If you use a white background, apply a gray border to your card instead of a drop shadow." 

![image](https://user-images.githubusercontent.com/51165119/117982361-edc02b00-b303-11eb-9991-8a0d9a010c31.png)

Reply:
I added a note with slightly different wording.